raiden/network/rpc/client.py

```diff
@@ -187,8 +187,8 @@ def new_filter(jsonrpc_client, contract_address, topics, from_block=None, to_blo
     if isinstance(to_block, int):
         to_block = hex(to_block)
     json_data = {
-        'fromBlock': from_block if from_block is not None else 'latest',
```

Comment:
Lefteris, but that's exactly why I took of explicit None check. It was a `is not None`, therefore, if the argument came as `''`, it would fall into the `true` part of the `if` (because empty string is not None, even being false). As it may come as empty string, or even as `0`, with my change, it will correctly fall into the `else` block, therefore, with the default argument of `earliest`, or _from_ the oldest block known.